src/logger.rs

```diff
@@ -61,6 +63,15 @@ fn should_log(record: &Record) -> bool {
 
 fn open_logfile() -> File {
     let mut path = std::env::current_dir().unwrap();
+    if cfg!(target_os = "macos") {
+        match get_instances_dir() {
+            Some(instance_path) => {
+                path = instance_path;
+                fs::create_dir_all(path.clone()).expect("Creation of instance directories failed.");
```

Comment:
Dito here - expect is no good, handle it or propagate it using `?`.

Reply:
Agree, but my hope is that you rewrite the logger instantiation so that the log file ends on a proper place on all platforms.